101-sql-vm-new-storage-ultrassd/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "Location for all resources."
+      },
+      "defaultValue": "East US 2"
```

Comment:
remove defaultValue or use resourceGroup().location

Reply:
The VNet need to be in the same location so I specify East US 2 since the main template default location is East US 2